dev-docs/bidders/resetdigital.md

```diff
@@ -20,30 +20,37 @@ media_types: banner, video
 sidebarType: 1
 ---
 
-### bid params
+### Note
+
+Prebid adapter for Reset Digital requires approval and account setup. Video is supported but requires a publisher supplied renderer at this time.
+
+### Bid Params
 
 {: .table .table-bordered .table-striped }
 
 | Name     | Scope    | Description | Example                            | Type     |
 |----------|----------|-------------|------------------------------------|----------|
-| `pubId` | required |    Publisher account id         | `'123pubId'` | `string` |
-| `siteID` | optional |    Publisher site id         | `'123siteId'` | `string` |
+| `pubId` | required |    Publisher Account ID provided by ResetDigital    | `'123pubId'` | `string` |
+| `siteID` | optional |    Publisher Site ID         | `'123siteId'` | `string` |
 | `zoneId` | optional |   Used for extra fields          | `{}` | `object` |
+| `zoneId.placementId` | optional | ID used for reporting purposes | `"<id>"`    | `string` |
```

Comment:
As we've discussed in separate email, forcing publishers to put parameters in a JSON object makes things awkward. We can't stop you from doing this, but we can ask nicely that you consider putting these fields at the top level rather than burying them in the zoneId object. Pretty please.

Reply:
Thank you for your feedback. Addressing this adjustment is next on our roadmap. Our immediate priority is to update only the documentation for a specific publisher based on our current setup, but we will make the requested changes in the next update.
In the meantime, we kindly ask for your approval of the current documentation update. The fix will follow shortly after.
Thank you for your understanding and cooperation.